scripts/globals/effects/aggressor.lua

```diff
@@ -11,6 +11,12 @@ require("scripts/globals/status");
 -----------------------------------
 
 function onEffectGain(target,effect)
+   -- Add Ranged Attack bonus.
+   local racc = effect:getPower();
+   if racc > 0 then

```

Comment:
you don't need to check for greater than zero, just do 
target:addMod(MOD_RACC, effect:getPower());
looks nicer on one line and is functionally no different